RecoVertex/VertexTools/interface/SequentialVertexFitter.h

```diff
@@ -226,6 +242,9 @@ class SequentialVertexFitter : public VertexFitter<N> {
   float theMaxShift;
   int theMaxStep;
 
+  float trackerBoundsRadius{112.};
```

Comment:
if you don't plan to make it configurable (as per PR description), at least preserve the original comment about having to take it from elsewhere.

Reply:
addressed at https://github.com/cms-sw/cmssw/pull/40479/commits/170b1938f837ef274be19a942ddccac4a83c89d0